main/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/bytebase/dbcost/client"
+	"github.com/bytebase/dbcost/client/aws"
+	"github.com/bytebase/dbcost/client/gcp"
+	"github.com/bytebase/dbcost/store"
+)
+
+var (
+	cloudProvider = make(map[store.CloudProvider]client.Client)
+	// renderEnvKey is set on render
+	renderEnvKey = "API_KEY_GCP"
+	apiKeyGCP    = ""
+	dirPath      = "./store/data"
+	fileName     = "rds.json"
+	filePath     = path.Join(dirPath, fileName)
+)
+
+func init() {
+	if os.Getenv("RENDER") == "true" {
```

Comment:
Should this be if API_KEY_GCP exists in the environment variable, fetch it? Otherwise, throw an error and ask users to set the environment variable.